docs/dqx/docs/reference.mdx

```diff
@@ -32,9 +34,9 @@ The following quality rules / functions are currently available:
 
 You can check implementation details of the rules [here](https://github.com/databrickslabs/dqx/blob/main/src/databricks/labs/dqx/col_functions.py).
 
-## Creating your own checks
+### Creating your own checks
 
-### Use sql expression
+**Use sql expression**
```

Comment:
why not `####` ? Is it because the doc engine breaks it into subpages?

Reply:
corrected, also updated other places to use #### instead of **